api/config/config_test.json

```diff
@@ -85,7 +85,8 @@
   },
   "moodle": {
     "host": "aoudev.nnlm.gov",
-    "credentialsKeyV2": "moodle-key-v2.txt"
+    "credentialsKeyV2": "moodle-key-v2.txt",
+    "lookbackPeriod": 30
```

Comment:
I think it would be clearer to collocate this with the other renewal settings instead, e.g.

```
access {
  renewal {
  ...
  lookbackPeriod: ..,
  trainingLookbackPeriod: ...
```

Reply:
Good call. I updated the config to reflect your suggestion.